relay/src/main.rs

```diff
@@ -35,9 +35,10 @@
 //!  - [`relay-base-schema`]: Basic types for Relay's API schema used across multiple services.
 //!  - [`relay-cabi`]: C-bindings for exposing functionality to Python.
 //!  - [`relay-cardinality`]: Metrics cardinality limiter.
+//!  - [`relay-cogs`]: Break down the cost of Relay by its features.
 //!  - [`relay-common`]: Common utilities and crate re-exports.
 //!  - [`relay-config`]: Static configuration for the CLI and server.
-//!  - [`relay-cogs`]: Break down the cost of Relay by its features.
+//!  - [`relay-conventions`]: Attribute definitions extracted from `sentry-conventions`.
```

Comment:
Could also link to the repo here.

```suggestion
//!  - [`relay-conventions`]: Attribute definitions extracted from [`sentry-conventions`](https://github.com/getsentry/sentry-conventions).
```

Reply:
Good idea, missed that.